doc/user/content/sql/create-source/mysql.md

```diff
@@ -0,0 +1,387 @@
+---
+title: "CREATE SOURCE: MySQL"
+description: "Connecting Materialize to a MySQL database"
+pagerank: 40
+menu:
+  main:
+    parent: 'create-source'
+    identifier: cs_mysql
+    name: MySQL
+    weight: 20
+---
+
+{{< private-preview />}}
+
+{{% create-source/intro %}}
+Materialize supports MySQL (5.7+) as a real-time data source. To connect to a
+MySQL database, you first need to tweak its configuration to enable
+[GTID-based binary log (binlog) replication](#change-data-capture), and then
+[create a connection](#creating-a-connection) in Materialize that specifies
+access and authentication parameters.
+{{% /create-source/intro %}}
+
+## Syntax
+
+{{< diagram "create-source-mysql.svg" >}}
+
+Field | Use
+------|-----
+_src_name_  | The name for the source.
+**IF NOT EXISTS**  | Do nothing (except issuing a notice) if a source with the same name already exists. _Default._
+**IN CLUSTER** _cluster_name_ | The [cluster](/sql/create-cluster) to maintain this source.
+**CONNECTION** _connection_name_ | The name of the MySQL connection to use in the source. For details on creating connections, check the [`CREATE CONNECTION`](/sql/create-connection/#mysql) documentation page.
+**FOR ALL TABLES** | Create subsources for all tables in all schemas upstream.
+**FOR SCHEMAS (** _schema_list_ **)** | Create subsources for specific schemas upstream.
```

Comment:
In mysql databases are the same as schemas, and it seems that their docs mostly use the term `database` when referring to table namespacing, so we might want to reword to highlight the ambiguous term.

Also one thing to call out is that `FOR ALL TABLES` will ignore the `mysql` schema/database, which contains a whole bunch of system tables, even though many mysql tutorials default to putting user-created tables in this database (annoyingly). We have no way to distinguish system tables and user-created tables within that database which is why we have to ignore it. We might want to include a note about this somewhere.